packages/webapp/src/store/api/apiSlice.ts

```diff
@@ -290,6 +294,23 @@ export const api = createApi({
       invalidatesTags: (_result, error) =>
         error ? [] : ['FarmAddon', 'Sensors', 'SensorReadings'],
     }),
+    getIrrigationPrescriptions: build.query<IrrigationPrescription[], void>({
+      query: () => `${irrigationPrescriptionsUrl}`,
+      async onQueryStarted(_id, { dispatch, queryFulfilled }) {
+        try {
+          // Replacement for invalidates tags 'Tasks'
```

Comment:
Since it kind of requires up to date tasks I did something in the api query that is a bit different to avoid concurrency issues, please check that out and tell me if you think it is similar to invalidating tags.

Reply:
Yeah agreed, but there is no Tasks tag currently. And that would be a big refactor I think

Reply:
Ohhh right! Okay this makes sense then, could you expand the comment a little bit and explain how this should be replaced once Tasks is migrated?